_includes/assets/js/indicatorModel.js

```diff
@@ -377,6 +377,16 @@ var indicatorModel = function (options) {
           }
         }
       },
+      getBackgroundColor = function(datasetIndex) {
+
+        var color = '#' + getColor(datasetIndex);
+        // offset if there is no headline data:
+        if(!this.hasHeadline) {
+          datasetIndex += 1;
+        }
+        // Either a solid color, or a pattern via the "patternomaly" library.
```

Comment:
Your code below is quite well self-documented so you don't need this comment (no need to mention the library used).

Tip: Best for your code to be self-documented instead of relying on comments

Reply:
Good advice, thanks. I tweaked it to add some safety code. Your comment made me realize my mentioning of the library was really just a useless attempt to document that it relies on a global variable on the "window" object. So I converted it into an explicit check for that variable.